src/components/vega-lite/index.tsx

```diff
@@ -3,16 +3,19 @@ import * as vega from 'vega';
 import * as vl from 'vega-lite';
 import {TopLevelExtendedSpec} from 'vega-lite/build/src/spec';
 import * as vegaTooltip from 'vega-tooltip';
+import {LOG_ERROR_CHANGE, LOG_WARNINGS_ADD, LogAction} from '../../actions/log';
+import {ActionHandler} from '../../actions/redux-action';
+import {localLogger} from '../../store/index';
```

Comment:
I don't think we should have a singleton local logger.  This makes the VegaLite unnecessary coupled.  Instead, each VegaLite component takes its own logger as input.